addon/helpers/split-grid-detail-footer.js

```diff
@@ -0,0 +1,24 @@
+import Ember from 'ember';
+
+/**
+ * @module helpers
+ */
+
+/**
+ * Lookup and output a split-grid row's cell value based on column's path
+ *
+ * @function split-grid-detail-footer
+ * @param    {object} options - The Ember-supplied bound options representing
```

Comment:
All of the other files in this repository use Ember.Object. Please change the occurrence for this property to be consistent with the rest of the codebase. If it is desired to move away from the "Ember." syntax create an issue to be worked on separately.